src/main/java/racingcar/model/CarFactory.java

```diff
@@ -0,0 +1,32 @@
+package racingcar.model;
+
+import racingcar.model.Car;
+
+import java.util.List;
+import java.util.Set;
+
+import static java.util.stream.Collectors.toList;
+
+public class CarFactory {
+
+    private static final int START_SCORE = 1;
+
+    public static List<Car> createCar(List<String> carNameList) {
+        validateDuplicateName(carNameList);
+
+        return carNameList.stream()
+                .map(carName -> {
+                    return new Car(carName, START_SCORE);
+                })
```

Comment:
람다식이 한줄로 끝난다면 아래와 같이 `{}`  블록과 return 문을 제거할 수 있어 가독성이 좋아져요. 🙂
```suggestion
                .map(carName -> new Car(carName, START_SCORE))
```